extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -108,10 +108,12 @@ void ListenHTTP::onSchedule(core::ProcessContext *context, core::ProcessSessionF
 
   bool randomPort = listeningPort == "0";
 
-  std::string authDNPattern;
+  std::string authDNPattern = ".*";
 
   if (context->getProperty(AuthorizedDNPattern.getName(), authDNPattern) && !authDNPattern.empty()) {
     logger_->log_debug("ListenHTTP using %s: %s", AuthorizedDNPattern.getName(), authDNPattern);
+  } else {
+    logger_->log_debug("Authorized DN Pattern not set or invalid, using default '%s' pattern", authDNPattern);
```

Comment:
shouldn't we set `authDNPattern` here to the default `".*"`? I think the property could contain the empty string value and overwrite the default at line 113

Reply:
Good catch, fixed in 780e0ce9d630ec0fd4214b676a2d19da5d840755